app/controllers/increase_evs_controller.rb

```diff
@@ -0,0 +1,10 @@
+class IncreaseEvsController < ApplicationController
+  def create
+    pokemon = Pokemon.find(params[:pokemon_id])
+    ev = EffortValue.find(params[:pokemon][:evs].to_i)
```

Comment:
You don't need `to_i` here. You should also be validating this value with strong_params since it doesn't come from the URL.